src/pages/guides/index.md

```diff
@@ -256,4 +256,14 @@ hmac.update(raw_request_body)
 if (request.header('x-adobe-signature') !== hmac.digest('base64')) {
   throw new Error('x-adobe-signature HMAC check failed')
 }
-```
\ No newline at end of file
+```
+
+## Quotas
+
+There is an upper limit on the number of registrations that you can create. Following table lists the applicable quotas - 
```

Comment:
Does console handle this case gracefully today? Does it display an error message?
I think it would be good to add that an appropriate error will be returned in case this limit is breached.

Reply:
I tested this on Console. It displays a generic error message - 
![image](https://user-images.githubusercontent.com/5928555/149782083-30fe09a2-c8f6-4103-9216-7975a464d05c.png)


Yes, it would be nice to inform the user about this limit being breached. I will create a ticket for it. We probably can work on it in our next wave of UI changes.